curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc9c4a6dd6d7dd3e6357.md

```diff
@@ -22,14 +23,12 @@ The content area is the innermost part of the box. It's the space that contains
 
 The padding is the area immediately after the content area. It's the space between the content area and the border of an element. With the padding you can add space around the content to improve it's readability. You can set different values for the top, right, bottom and left padding with the `padding` property.  
 
-This is an example with the `padding` shorthand property, where we set the top padding to fifteen pixels, the left padding to five pixels, the bottom padding to two pixels and the left padding to eight pixels:  
+This is an example with the `padding` shorthand property, where we set the top padding to fifteen pixels, the right padding to five pixels, the bottom padding to two pixels and the left padding to eight pixels:  
```

Comment:
Here is the issue for this change 

https://github.com/freeCodeCamp/freeCodeCamp/issues/60549